pyrit/orchestrator/single_turn/question_answer_benchmark_orchestrator.py

```diff
@@ -184,10 +185,22 @@ async def run_attacks_async(  # type: ignore[override]
             question_answering_entries (list[QuestionAnsweringEntry]): List of question answering entries to process.
             prepended_conversations (list[PromptRequestResponse], Optional): The conversations to prepend to each
                 attack.
+            question_repeat_count (int): allows for repetition of QuestionAnsweringEntry objects
```

Comment:
```suggestion
            repetitions (int): each question will be sent to the target this many times
```
"question" is implicit given the orchestrator, so maybe "repetitions" is better? Come to think of it, that is probably something we could have on all orchestrators but don't worry about that.

Reply:
Good point, renamed